projects/plugins/jetpack-boost/bin/version-files.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
```

Comment:
Shouldn't be needed in the monorepo. We already have a similar tool.

Reply:
https://github.com/Automattic/jetpack/pull/20512/commits/20530e155c219bc6baf3899c5469f03dabbe4025